shared/crypto/src/main/scala/io/iohk/atala/shared/crypto/Apollo.scala

```diff
@@ -0,0 +1,105 @@
+package io.iohk.atala.shared.crypto
+
+import io.iohk.atala.shared.models.HexString
+import zio.*
+
+import scala.util.Try
+
+trait Apollo {
```

Comment:
We also need this on the prism-node.
Would be great if we could move as max as possible of this logic to Apollo

Reply:
we should  move it to apollo for adding more keys it will be better